change_logs/ChangeLog.html

```diff
@@ -50,6 +50,7 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
 <li><b>Enhancements</b></li>
 <ul>
 <li><font color="red">Improved the Sphere texture mapping that now applies UV mapping.</font></li>
+<li><font color="red">Encourage the use of quadratic attenuation for PointLight and SpotLight nodes by displaying a warning in case the 'attenuation' field does not provide realistic lighting.</font></li>
```

Comment:
```suggestion
<li><font color="red">Encourage the use of realistic quadratic attenuation for PointLight and SpotLight nodes by displaying a warning in case the 'attenuation' field is not quadratic.</font></li>
```